packages/story-editor/src/components/canvas/emptyStateLayer.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import {
+  BUTTON_TYPES,
+  BUTTON_VARIANTS,
+  Button,
+  Icons,
+} from '@web-stories-wp/design-system';
+
+/**
+ * Internal dependencies
+ */
+import { __ } from '@web-stories-wp/i18n';
+import { useRightClickMenu } from '../../app';
+import useStory from '../../app/story/useStory';
+import { DEFAULT_PAGE_BACKGROUND_COLOR } from '../../elements/utils/createPage';
+import { Layer, PageArea } from './layout';
+import { shallowEqual } from '@web-stories-wp/react';
+
+const DisplayPageArea = styled(PageArea)`
+  position: absolute;
+  z-index: 1;
+`;
+
+const EmptyStateMessage = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+export const isEmptyStory = (pages) => {
+  if (pages.length === 0) {
+    return false;
+  }
+
+  const hasOnlyOnePage = pages.length === 1;
+  const hasNoExtraElements = pages[0].elements.length === 1;
+  const hasNoBackgroundMedia = !('resource' in pages[0].elements[0]);
+  const hasDefaultBackgroundColor = shallowEqual(
+    DEFAULT_PAGE_BACKGROUND_COLOR.color,
+    pages[0].backgroundColor.color
+  );
+
+  return (
+    hasOnlyOnePage &&
+    hasNoExtraElements &&
+    hasNoBackgroundMedia &&
+    hasDefaultBackgroundColor
+  );
+};
+
+function EmptyStateLayer() {
+  const onOpenMenu = useRightClickMenu((value) => value.onOpenMenu);
+  const { pages } = useStory(({ state: { pages } }) => ({
+    pages,
+  }));
+
+  if (!isEmptyStory(pages)) {
+    return null;
+  }
+
+  return (
+    <Layer onContextMenu={onOpenMenu}>
+      <DisplayPageArea withSafezone={false}>
+        <EmptyStateMessage>
+          <Button
+            type={BUTTON_TYPES.SECONDARY}
+            variant={BUTTON_VARIANTS.CIRCLE}
+            onClick={onOpenMenu}
+          >
+            <Icons.Media />
+          </Button>
```

Comment:
Hmm, I can't focus (keyboard only) on anything on the canvas from a state like this (deselected everything):
![image](https://user-images.githubusercontent.com/21036224/150380105-522d8326-ecd4-4a89-9ae8-bed5d1429914.png)
^ tested on main.
Can you select again the element?

The only thing I can focus on is this empty state button: 
![image](https://user-images.githubusercontent.com/21036224/150380371-7561ac25-c100-4624-a60a-c59a10cffd0d.png)

Reply:
Ah yes, it does participate in the tab order, just not where I expected it to be. But that's fine, as long as it's there.

However, it still needs:

* `aria-labelledby` with a reference to the text element below it
* * or maybe it should actually be `aria-describedby` because it's not the name of the button as much as a description of what it does? If so, it should probably have an `aria-label` too, maybe `aria-label="Add content"` or similar? We don't really utilize descriptions much, but maybe we should look more into that.
* `aria-haspopup="true"`
* `aria-expanded="true/false"`

And yes, there also seems to be a bug around elements on canvas not being selectable through tabbing, which is definitely a problem. Is this a known bug? If not, feel free to file it.

Reply:
I've added `aria-describedby`,`label` and `aria-haspopup`, not sure about `aria-expanded` because it's already saying "Context menu for the selected element, dialog, with X items" when you open it, so "Expanded" wouldn't add much info here?

Reply:
There are different tools relying on `aria-*` properties, not just screen readers. And different screen readers do things differently. So I still think we should follow the recommendations and add the `aria-expanded` property, even if we can't actually tell if it makes a difference or not.

Reply:
Alright https://github.com/GoogleForCreators/web-stories-wp/pull/10261/commits/3921f63071a5e36f79bbef295e2c0e0622d96a2c